### ISSUE TITLE
Implement new LF-wide header

### ISSUE THREAD

**Issue by thetwopct** [LF-Engineering/lfevents#686]
LF are updating their header/navigation and LF events needs to follow suit:

![2022-09-08-113753@2x](https://user-images.githubusercontent.com/10615884/189035498-17ebd02e-425a-45b4-839b-cefd77eac96f.jpg)

**Comment by thetwopct** [comment]
Feedback required: 
- Event Splash page has been updated as it shares the header with home page - I need feedback if:

1. the current implementation is acceptable or the layout should be more custom
2. the top blue LF bar should be included on these pages or not

![2022-09-13-010657](https://user-images.githubusercontent.com/10615884/189725215-c5164690-a427-4b66-9895-52c429123d25.jpg)

Steps to do on Live:
- [ ] Navigation - Remove LF Sites menu
- [ ] Navigation - Remove Nested Pages menu (unless still in use?)
- [ ] Change button colors on Sponsor, Newsletter and Contact forms

**Comment by thetwopct** [comment]
SSO successfully tested via dev site:

![2022-09-13-125845@2x](https://user-images.githubusercontent.com/10615884/189821315-f44d1cdc-e95e-47c7-a7e7-9b1d03d3b0db.jpg)

![2022-09-13-125908@2x](https://user-images.githubusercontent.com/10615884/189821358-5fdf2906-7248-40f7-a0bd-15552134fc92.jpg)

**Comment by cjyabraham** [comment]
I will deploy this shortly, however, I'll leave this issue open as there are a few remaining comments on the figma and I imagine we may want to fine-tune things in response to how they've done it on the LF site.

**Comment by cjyabraham** [comment]
Looks like these sites have been deployed so far. Each seems to have handled the details slightly differently:
- https://lfx.linuxfoundation.org/
- https://training.linuxfoundation.org/

**Comment by thetwopct** [comment]
I've ran the site through W3C, WAVE, and Lighthouse and updated a few things to reflect that, as well as actioned the changes in Figma. I have pushed this up to dev for you to test.

I've checked out the other sites and am a bit confused as to what to follow and not....

Some thoughts:

- All the sites except ours use purple in the mobile menu so perhaps we should follow suit
- The use of the bright pink as CTA seems to be used on the LF parent site and sub-sets so we could possibly utilise that where it makes sense

**Comment by cjyabraham** [comment]
A few comments:
- the Subscribe button in the footer form now turns dark-blue on hover, the same color as the whole footer, so disappears essentially on hover. I assume this is not what you wanted. I've implemented [a fix](https://github.com/LF-Engineering/lfevents/commit/b90c89a8c80906d0abe6774ee907939c8ab59d76).
- I don't like the purple the other sites use so don't think we should use it unless someone says we need to. Same with the bright pink (which also doesn't meet WCAG contrast requirements).

I've deployed the updates.
